### DISCUSSION TITLE
[Bug] Knowledge base can only upload PDF, PPT and other files, cannot upload images

### DISCUSSION THREAD

**Discussion by LuYouxi** [lobehub/lobehub#12069] [Self Hosting] [answered]
### 📦 部署环境

Docker

### 📌 软件版本

1.34.5

### 💻 系统环境

Ubuntu

### 🌐 浏览器

Edge

### 🐛 问题描述

本地数据库版本，生产环境布置，不能上传图片，但是可以上传PDF、PPT等文件。

[a comment by an automated account is not shown]

**Comment by yuntian123456** [comment]
使用的nginx proxy manager作域名反代的么。。。

[a comment by an automated account is not shown]

**Comment by LuYouxi** [comment]
> 使用的nginx proxy manager作域名反代的么。。。

是的

[a comment by an automated account is not shown]

**Comment by yuntian123456** [comment]
你还可以试试，不能上传png,jpg, gif格式的，但可以上传bmp格式的。。。
经过我的测试，发现是nginx proxy manager导致的。**原因未知**。
换成原版的nginx，所有配置一样的情况下，问题就解决了。

[a comment by an automated account is not shown]

**Comment by LuYouxi** [comment]
> 你还可以试试，不能上传png,jpg, gif格式的，但可以上传bmp格式的。。。 经过我的测试，发现是nginx proxy manager导致的。**原因未知**。 换成原版的nginx，所有配置一样的情况下，问题就解决了。

确实很莫名奇妙，不知道在使用nginx proxy manager下如何解决。

[a comment by an automated account is not shown]

**Comment by LuYouxi** [comment]
> 你还可以试试，不能上传png,jpg, gif格式的，但可以上传bmp格式的。。。 经过我的测试，发现是nginx proxy manager导致的。**原因未知**。 换成原版的nginx，所有配置一样的情况下，问题就解决了。

请问在已经有NGINX proxy mannager的情况下，怎么使用原版的NGINX部署，使其两者不冲突。

[a comment by an automated account is not shown]

**Comment by LuYouxi** [comment]
> 你还可以试试，不能上传png,jpg, gif格式的，但可以上传bmp格式的。。。 经过我的测试，发现是nginx proxy manager导致的。**原因未知**。 换成原版的nginx，所有配置一样的情况下，问题就解决了。

还有如何使用原版的NGINX进行部署。我是用Docker compose进行部署的。

[a comment by an automated account is not shown]

**Comment by CoREse** [comment, marked as answer]
找到问题所在啦，见这里 #5254

[a comment by an automated account is not shown]
